engine/emoji-picker

```diff
@@ -16,8 +16,8 @@
 #  You should have received a copy of the GNU General Public License
```

Comment:
Creating them from the original SVG icons should be fine for the moment.

I don’t know where the original icon comes from, I didn’t create it, it was already there when
I started  working on ibus-typing-booster. I would like to replace it with something nicer looking,
maybe a rocket? ( https://en.wikipedia.org/wiki/Booster_(rocketry) ) But I have
never designed icons either. So I think for the moment creating .png files from the original .svg
should be OK.